service/src/main/java/uk/gov/hmcts/reform/fpl/model/Solicitor.java

```diff
@@ -13,8 +14,13 @@
     private final String dx;
     @NotBlank(message = "Enter the solicitor's full name")
     private final String name;
+
     @NotBlank(message = "Enter the solicitor's email")
+    @Email(
+        message = "Enter a valid email address",
```

Comment:
Current validation on CaseData tends to include empty validation i.e if empty then error message = "Enter an email". If not empty but incorrect then we can read "Enter a valid address". Do we need to consider the same for consistency?